<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana_disponibilidad</class>
 <widget class="QDialog" name="ventana_disponibilidad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ventana de disponibilidad</string>
  </property>
  <widget class="QLabel" name="fondo_disponibilidad">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>771</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>fondo_disponibilidad_fixed.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="contenedor">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>901</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">/* QTabWidget */
QTabWidget::pane {
    border: 1px solid #b0b0b0;
    border-radius: 8px;
    background-color: #ffffff; /* Fondo blanco para el área de contenido */
}

/* Pestañas grises */
QTabBar::tab {
    background-color: #e0e0e0;
    color: #333333;
    padding: 8px 16px;
    border: 1px solid #c0c0c0;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
    margin: 2px;
}

/* Pestaña seleccionada */
QTabBar::tab:selected {
    background-color: #d0d0d0; /* Gris claro para la pestaña seleccionada */
    border: 1px solid #a0a0a0;
    color: #000000;
}

/* Pestaña no seleccionada */
QTabBar::tab:!selected {
    background-color: #f0f0f0;
    color: #555555;
}

/* Hover sobre pestaña */
QTabBar::tab:hover {
    background-color: #d8d8d8;
    border: 1px solid #b0b0b0;
}

/* Botones de desplazamiento (cuando hay muchas pestañas) */
QTabBar QToolButton {
    background-color: #f0f0f0;
    border: none;
    color: #555555;
}

/* Estilo de los botones al hacer hover */
QTabBar QToolButton:hover {
    background-color: #d0d0d0;
}

/* Estilo de la pestaña cuando está deshabilitada */
QTabBar::tab:disabled {
    color: #888888;
    background-color: #e8e8e8;
    border: 1px solid #d0d0d0;
}

/* Ajustes de tamaño */
QTabBar::tab {
    min-width: 100px;
    min-height: 32px;
}
</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tipo de habitación</string>
    </attribute>
    <widget class="QLabel" name="instruccion_disponibilidad_tipo_habitacion">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Por favor, ingrese el &lt;span style=&quot; font-weight:600;&quot;&gt;tipo de habitación&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;a consultar como disponible:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="entrada_consultar_disponibilidad_tipo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>521</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">Ejemplo: Suite Presidencial</string>
     </property>
     <property name="placeholderText">
      <string notr="true"/>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="logo_casa">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>10</y>
       <width>111</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>logo_casa.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_verificar_disponibilidad_tipo">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>70</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">/* QPushButton */
QPushButton {
    background-color: #78c897; /* Verde suave */
    color: #ffffff;
    border: 2px solid #6aae84;
    border-radius: 8px;
    padding: 8px 16px;
    font-size: 14px;
}

/* Hover (cuando el cursor está sobre el botón) */
QPushButton:hover {
    background-color: #6aae84; /* Verde un poco más oscuro para hover */
    border: 2px solid #5f9676;
}

/* Presionado (cuando se hace clic en el botón) */
QPushButton:pressed {
    background-color: #5f9676; /* Verde más oscuro para clic */
    border: 2px solid #4e8264;
    color: #e0e0e0;
}

/* Botón deshabilitado */
QPushButton:disabled {
    background-color: #c0c0c0; /* Gris claro para deshabilitado */
    color: #888888;
    border: 2px solid #b0b0b0;
}

/* Botón con foco (cuando está seleccionado con teclado o tab) */
QPushButton:focus {
    outline: none;
    border: 2px solid #80bfff; /* Azul suave para indicar foco */
}
</string>
     </property>
     <property name="text">
      <string>Verificar disponibilidad</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QScrollArea" name="scroll_habitacione_disponibles_tipo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>861</width>
       <height>191</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>859</width>
        <height>189</height>
       </rect>
      </property>
      <widget class="QListWidget" name="lista_habitaciones_disponibles_tipo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>831</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    font: 12pt &quot;Century Gothic&quot;;
    background: transparent;
    border: none;
}

QLabel {
    font-weight: bold;  /* Aplica negrita a todos los QLabel */
}

QListWidget::item:hover {
    background-color: #90EE90;  /* Verde claro */
}

QListWidget::item:selected {
    background-color: #008000;  /* Verde oscuro */
    color: white;  /* Texto blanco */
}

</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Precio de habitación</string>
    </attribute>
    <widget class="QLabel" name="logo_precio">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>30</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>logo_precio.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="instruccion_disponibilidad_precio_habitacion">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Por favor, ingrese el&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt; precio&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; de las habitaciones que desea &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;consultar como disponibles:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="entrada_consultar_precio">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Ejemplo: 250</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_verificar_disponibilidad_precio">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>80</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">/* QPushButton */
QPushButton {
    background-color: #78c897; /* Verde suave */
    color: #ffffff;
    border: 2px solid #6aae84;
    border-radius: 8px;
    padding: 8px 16px;
    font-size: 14px;
}

/* Hover (cuando el cursor está sobre el botón) */
QPushButton:hover {
    background-color: #6aae84; /* Verde un poco más oscuro para hover */
    border: 2px solid #5f9676;
}

/* Presionado (cuando se hace clic en el botón) */
QPushButton:pressed {
    background-color: #5f9676; /* Verde más oscuro para clic */
    border: 2px solid #4e8264;
    color: #e0e0e0;
}

/* Botón deshabilitado */
QPushButton:disabled {
    background-color: #c0c0c0; /* Gris claro para deshabilitado */
    color: #888888;
    border: 2px solid #b0b0b0;
}

/* Botón con foco (cuando está seleccionado con teclado o tab) */
QPushButton:focus {
    outline: none;
    border: 2px solid #80bfff; /* Azul suave para indicar foco */
}
</string>
     </property>
     <property name="text">
      <string>Verificar disponibilidad</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QScrollArea" name="scroll_habitaciones_disponibles_precio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>861</width>
       <height>181</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>859</width>
        <height>179</height>
       </rect>
      </property>
      <widget class="QListWidget" name="lista_habitaciones_disponibles_precio">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>10</y>
         <width>841</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    font: 12pt &quot;Century Gothic&quot;;
    background: transparent;
    border: none;
}

QLabel {
    font-weight: bold;  /* Aplica negrita a todos los QLabel */
}

QListWidget::item:hover {
    background-color: #90EE90;  /* Verde claro */
}

QListWidget::item:selected {
    background-color: #008000;  /* Verde oscuro */
    color: white;  /* Texto blanco */
}

</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Número de habitación</string>
    </attribute>
    <widget class="QLabel" name="logo_numero_habitacion">
     <property name="geometry">
      <rect>
       <x>-70</x>
       <y>0</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>logo_numero_habitacion.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="instruccion_disponiblidad_numero_habitacion">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>391</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Por favor, ingrese el&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt; número de la habitación&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; que desea &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;consultar como disponible&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="entrada_consultar_disponibilidad_numero">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Ejemplo: 102</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_verificar_disponibilidad_numero">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>80</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">/* QPushButton */
QPushButton {
    background-color: #78c897; /* Verde suave */
    color: #ffffff;
    border: 2px solid #6aae84;
    border-radius: 8px;
    padding: 8px 16px;
    font-size: 14px;
}

/* Hover (cuando el cursor está sobre el botón) */
QPushButton:hover {
    background-color: #6aae84; /* Verde un poco más oscuro para hover */
    border: 2px solid #5f9676;
}

/* Presionado (cuando se hace clic en el botón) */
QPushButton:pressed {
    background-color: #5f9676; /* Verde más oscuro para clic */
    border: 2px solid #4e8264;
    color: #e0e0e0;
}

/* Botón deshabilitado */
QPushButton:disabled {
    background-color: #c0c0c0; /* Gris claro para deshabilitado */
    color: #888888;
    border: 2px solid #b0b0b0;
}

/* Botón con foco (cuando está seleccionado con teclado o tab) */
QPushButton:focus {
    outline: none;
    border: 2px solid #80bfff; /* Azul suave para indicar foco */
}
</string>
     </property>
     <property name="text">
      <string>Verificar disponibilidad</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QScrollArea" name="scroll_habitaciones_disponibles_numero">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>142</y>
       <width>871</width>
       <height>181</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>869</width>
        <height>179</height>
       </rect>
      </property>
      <widget class="QListWidget" name="lista_habitaciones_disponibles_numero">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>851</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    font: 12pt &quot;Century Gothic&quot;;
    background: transparent;
    border: none;
}

QLabel {
    font-weight: bold;  /* Aplica negrita a todos los QLabel */
}

QListWidget::item:hover {
    background-color: #90EE90;  /* Verde claro */
}

QListWidget::item:selected {
    background-color: #008000;  /* Verde oscuro */
    color: white;  /* Texto blanco */
}

</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Todas las habitaciones disponibles</string>
    </attribute>
    <widget class="QLabel" name="logo_disponible">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>70</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>logo_disponible_1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="logo_disponible_habitacion">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>-50</y>
       <width>191</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>logo_disponible_2.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="instruccion_disponibilidad_estado">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>-10</y>
       <width>721</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;¿Deseas consultar&lt;span style=&quot; font-weight:600;&quot;&gt; todas las habitaciones&lt;/span&gt; que están disponibles actualmente?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_verificar_disponibilidad_estado">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">/* QPushButton */
QPushButton {
    background-color: #78c897; /* Verde suave */
    color: #ffffff;
    border: 2px solid #6aae84;
    border-radius: 8px;
    padding: 8px 16px;
    font-size: 14px;
}

/* Hover (cuando el cursor está sobre el botón) */
QPushButton:hover {
    background-color: #6aae84; /* Verde un poco más oscuro para hover */
    border: 2px solid #5f9676;
}

/* Presionado (cuando se hace clic en el botón) */
QPushButton:pressed {
    background-color: #5f9676; /* Verde más oscuro para clic */
    border: 2px solid #4e8264;
    color: #e0e0e0;
}

/* Botón deshabilitado */
QPushButton:disabled {
    background-color: #c0c0c0; /* Gris claro para deshabilitado */
    color: #888888;
    border: 2px solid #b0b0b0;
}

/* Botón con foco (cuando está seleccionado con teclado o tab) */
QPushButton:focus {
    outline: none;
    border: 2px solid #80bfff; /* Azul suave para indicar foco */
}
</string>
     </property>
     <property name="text">
      <string>Verificar disponibilidad</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QScrollArea" name="scroll_habitaciones_disponibles_estado">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>162</y>
       <width>881</width>
       <height>161</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>879</width>
        <height>159</height>
       </rect>
      </property>
      <widget class="QListWidget" name="lista_habitaciones_disponibles_estado">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>851</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    font: 12pt &quot;Century Gothic&quot;;
    background: transparent;
    border: none;
}

QLabel {
    font-weight: bold;  /* Aplica negrita a todos los QLabel */
}

QListWidget::item:hover {
    background-color: #90EE90;  /* Verde claro */
}

QListWidget::item:selected {
    background-color: #008000;  /* Verde oscuro */
    color: white;  /* Texto blanco */
}

</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_volver_al_menu">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>680</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
}</string>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ayuda_disponibilidad">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>680</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
}</string>
   </property>
   <property name="text">
    <string>Ayuda</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
